roles/common/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+
+- name: Log the OS name, version and release
```

Comment:
I think using the setup module might be nice here.  I *think* this should work.

```
-name: Log distro information
 setup:
     filter: "ansible_distribution_*"
```

That should print all ansible facts matching that filter pattern.